### PR TITLE
[#175227073] Add Token Name Update command

### PR DESCRIPTION
**Pull request by AleDore**
This PR introduce a new command used to set/update Token Name custom attribute on B2C extension app.

### CODE REVIEW

File: src/commands/users/update_user_token_name.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      chalk.blueBright.bold("Running"),
+      {
+        stdout: true
+      }
+    );
+
+    const servicePrincipalCreds = {
+      clientId: getRequiredStringEnv("SERVICE_PRINCIPAL_CLIENT_ID"),
+      secret: getRequiredStringEnv("SERVICE_PRINCIPAL_SECRET"),
+      tenantId: getRequiredStringEnv("SERVICE_PRINCIPAL_TENANT_ID")
```
Comment (gunzip) [inline src/commands/users/update_user_token_name.ts:53]: that would be nice to login using the azure client or with username and password instead (so this command can be run without sharing the `SERVICE_PRINCIPAL_SECRET` of a sp with very high privileges).

File: src/utils/azure.ts [hunk 2]
```diff
@@ -162,3 +166,29 @@ export const hasCosmosConnection = async (
...
+  readonly secret: string;
+  readonly tenantId: string;
+}
+
+export function getGraphRbacManagementClient(
+  adb2cCreds: IServicePrincipalCreds
+): TaskEither<Error, GraphRbacManagementClient> {
+  return tryCatch(
+    () =>
+      msRestNodeAuth.loginWithServicePrincipalSecret(
```
Comment (gunzip) [inline src/utils/azure.ts:181]: maybe `loginWithUserAndPassword` or with az client login ?

File: src/commands/users/update_user_token_name.ts (hunk 1 above)
Comment (AleDore) [reply src/commands/users/update_user_token_name.ts]: Added in [b920954](https://github.com/pagopa/io-ops/pull/128/commits/b920954548320f3fb5b39bc7e52b4e06910be43a)

File: src/utils/azure.ts (hunk 2 above)
Comment (AleDore) [reply src/utils/azure.ts]: Added in [b920954](https://github.com/pagopa/io-ops/pull/128/commits/b920954548320f3fb5b39bc7e52b4e06910be43a)

**Comment by AleDore** [comment]
@gunzip @francescopersico : Can we merge this ?

**Comment by AleDore** [comment]
> @gunzip @francescopersico : Can we merge this ?

ping :)

**Pull request merged**